ansible/configs/open-environment-azure-subscription/destroy_env.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Cleanup Azure Subscription Based Sandbox
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  become: false
+  environment:
+    AZURE_CONFIG_DIR: "{{ output_dir }}/.azure-{{ project_tag }}"
```

Comment:
Any reason this is a hidden directory within the output dir?

Reply:
it takes the place of ~/.azure